app/controllers/track.controller.js

```diff
@@ -148,4 +134,59 @@ exports.update = async (req, res) => {
           message: err.message + 'with id: ' + req.params.trackId
         })
       })
-  }
\ No newline at end of file
+  }
+
+// Delete a Track from specific album
+exports.delete = async (req, res) => {
+  const id = req.params.id;
+  const albumId = req.params.albumId;
+  Track.destroy({
+    where: { 
+      albumId: { [Op.like]: `%${albumId}%` },
```

Comment:
should this be equal instead of like?

Reply:
actually I thought about that but I didn't see that function/method in Sequilize (Op.equal), did not try it but I can see Op.like works fine

Reply:
https://sequelize.org/docs/v6/core-concepts/model-querying-basics/

Looks like there is a op.eq operator. If like works thats fine too then!